src/components/checks/[share_token]/practice/NavigationButtons.tsx

```
@@ -0,0 +1,18 @@
+'use client'
+
+import { ButtonHTMLAttributes } from 'react'
+import { usePracticeStore } from '@/src/components/checks/[share_token]/practice/PracticeStoreProvider'
+import { Button } from '@/src/components/shadcn/button'
+import { cn } from '@/src/lib/Shared/utils'
+
+export function PracticeNavigationNextButton({ ...props }: {} & ButtonHTMLAttributes<HTMLButtonElement>) {
+  const { nextQuestion } = usePracticeStore((store) => store)
+
+  return <Button {...props} onClick={() => nextQuestion(true)} children='Next' className={cn('', props.className)} />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)